### ISSUE TITLE
WPCS 0.14 disallows aligned calls to add_action(), add_filter(), etc

### ISSUE THREAD

**Issue by johnbillion** [WordPress-Coding-Standards/WordPress-Coding-Standards#1224]
The following code shows consecutive calls to `add_action()` which have their callback parameters aligned for readability. This passed WPCS sniffs prior to version 0.14, but now fails with errors:

```php
add_action( 'foo',      'bar' );
add_action( 'long_foo', 'baz' );
```

The error reported is:

```
Expected 1 space after comma in function call; 6 found (Generic.Functions.FunctionCallArgumentSpacing.TooMuchSpaceAfterComma)
```

I can't immediately find which change introduced this. While I can see that the sniff error is valid, I think this is something that should be allowed. Many themes, plugins, and indeed WordPress core align their hook callback functions to improve readability.

Is this a bug? If not, should this be considered an allowed pattern in WPCS?

**Comment by JDGrimes** [comment]
This was an intentional change, but I am aware that this pattern exists, and I have even used this form of alignment for function calls myself on some occasions. One could just disable the `TooMuchSpaceAfterComma` error, but then excess space that wasn't there for the purpose of alignment wouldn't get flagged either.

**Comment by GaryJones** [comment]
Definitely not a bug, and that pattern, even if used by some (I wouldn't say "many") projects, are breaking their own code standards. I agree that it can increase readability though.

**Comment by jrfnl** [comment]
@johnbillion Can this issue be closed ?
